odoo/tools/test_reports.py

```diff
@@ -74,7 +74,7 @@ def try_report(cr, uid, rname, ids, data=None, context=None, our_module=None, re
 def try_report_action(cr, uid, action_id, active_model=None, active_ids=None,
                 wiz_data=None, wiz_buttons=None,
                 context=None, our_module=None):
```

Comment:
By `our_module` I meant this, if there's only one caller we could just update `action_id` to include the module name (if that's not already the case).